### DISCUSSION TITLE
Your benefits will be available soon.

### DISCUSSION THREAD

**Discussion by Nem-IWIS** [community/community#144765] [GitHub Education]
### Hi everyone,

Good morning everyone.
I wanted to renew my expiring Github student offer. I have lost my Github student benefits as well as Copilot. My request has been in the following status for 3 days. "Your request was approved. Your benefits will be available soon."
Can you tell me when I will have access to the status of student, it becomes handicapping in my work and makes me waste time.

<img width="311" alt="Capture d’écran 2024-11-16 à 11 52 35" src="https://github.com/user-attachments/assets/0a08fadb-fdae-4ed5-9363-03a933774214">
Thank you very much, have a good day.

**Comment by davevad93** [comment]
Hi @Nem-IWIS ,

When the benefits become available, the status label will turn purple and change to:

- `Congrats! Your request for @Nem-IWIS was approved and your benefits are now available.`

The waiting time may vary depending on different factors, such as the volume of requests; it usually takes a few days, but sometimes it can take a little longer, even a couple of weeks.

For further reference you can look at these discussions: 
#132790 , #131665 , #132470 , #133531, #134279, #135849, #136887, #138721, #139268, #140287

And also the [Student Developer Pack Application FAQs](https://github.com/orgs/community/discussions/111352) :

- [How long after I submit my application will it be approved?](https://github.com/orgs/community/discussions/111352#how-long-after-i-submit-my-application-will-it-be-approved)
- [How long after I’ve been approved will I receive my academic benefits?](https://github.com/orgs/community/discussions/111352#how-long-after-ive-been-approved-will-i-receive-my-academic-benefits)

**Reply by Soni-Dheer** [reply]
It's already been 3 days , is there any way it could be done faster , because the project date is coming closer and closer and I didn't planned for this delay .

**Reply by davevad93** [reply]
Unfortunately no, because GitHub no longer provides manual reviews, you have to wait! 👍

**Comment by AshokChaudhary1** [comment]
Sir please accept my request, My exams are coming in December, please accept it soon, I have to do a lot of projects 
![IMG_20241116_190253](https://github.com/user-attachments/assets/b4f50ad0-da65-4c96-8087-363445a0be4a)


Thank you sir 🙏

**Comment by AdityaPandey-DEV** [comment]
do two factor authentication
